lib/wings/model_transformer.rb

```diff
@@ -135,6 +135,8 @@ def member_ids
         ordered_member_ids = pcdm_object.try(:ordered_member_ids)
         return ordered_member_ids if ordered_member_ids.present?
         pcdm_object.try(:member_ids)
+      rescue StandardError => _err
```

Comment:
@no-reply - I added the rescue StandardError block to fix a failed test.  I forgot which test failed before.  I removed it to confirm and now all tests still passed.  I think we don't need to catch anything there.

Reply:
@no-reply - One test failed for ruby2-6-2-valkyrie because I removed that rescue block.  
FrozenError:
       can't modify frozen ActiveFedora::Aggregation::ListSource::GeneratedResourceSchema
I added it back but this time catch FrozenError instead of StandardError.  The test passed now.  Could you review it again?  Thanks